packages/core/src/progress-circle/progress-circle.element.scss

```diff
@@ -0,0 +1,109 @@
+@import './../styles/tokens/generated/index';
+@import './../styles/mixins/mixins';
+
+:host {
+  --ringcolor: #{$cds-token-color-neutral-200};
+  --fillcolor: #{$cds-token-color-action-800};
+  font-family: inherit;
+  font-size: inherit;
+  color: inherit;
+  display: block;
+}
+
+svg {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.private-host {
+  @include center-content('block');
+  @include equilateral(100%);
+
+  .backstroke {
+    stroke: var(--ringcolor);
+  }
+
+  .arcstroke {
+    stroke: var(--fillcolor);
+    transition: stroke-dashoffset 300ms ease-in-out;
+  }
+}
+
+:host,
+:host([size='sm']) {
+  @include equilateral(#{$cds-token-space-size-7});
+}
+
+:host([size='md']) {
+  @include equilateral(#{$cds-token-space-size-9});
+}
+
+:host([size='lg']) {
+  @include equilateral(#{$cds-token-space-size-11});
+}
+
+:host([size='xl']) {
+  @include equilateral(#{$cds-token-space-size-12});
+}
+
+:host([size='xxl']) {
+  // 64px
+  @include equilateral(calc(#{$cds-token-space-size-13} - #{$cds-token-space-size-5}));
+}
+
+:host([value='0']) .fillstroke,
+:host([value='100']) .fillstroke {
```

Comment:
I need to differentiate between the fillstroke and the background stroke Because when both are visible at 100% things get really grainy and funky looking. At 0% the fill stroke needs to not be visible.